### PR TITLE
NodeStrategy supports node name argument.

### PR DESCRIPTION
**Pull request by fujitatomoya**
closes https://github.com/ros2/ros2/issues/1614

### CODE REVIEW

**Review by fujitatomoya — COMMENTED** [review]
Since this is not tested yet, i will keep this as draft.

but if you have any objections to have `node_name` argument in `NodeStrategy`, please share your thoughts. I do not really see the downside for this parameter exntension.

**Comment by fujitatomoya** [comment]
Pulls: ros2/ros2cli#941
Gist: https://gist.githubusercontent.com/fujitatomoya/81fe8695f0df1f4dde75155a6d12ba30/raw/b83bb9f7cc34fe9fd1a22c17bac34b2dfd316e54/ros2.repos
BUILD args:  --packages-above-and-dependencies ros2cli ros2topic
TEST args:  --packages-above ros2cli ros2topic
ROS Distro: rolling
Job: ci_launcher
ci_launcher ran: https://ci.ros2.org/job/ci_launcher/14732
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/22030/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16289/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1549/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22764/)

**Comment by fujitatomoya** [comment]
@clalancette @ahcorde can you review this? i do not think there is any downside for this.

File: ros2topic/ros2topic/verb/echo.py [hunk 1]
```diff
@@ -107,6 +107,9 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--include-message-info', '-i', action='store_true',
             help='Shows the associated message info.')
+        parser.add_argument(
+            '-n', '--node-name', type=str, default=None,
+            help='Name of the created echoing node')
```
Comment (clalancette) [inline ros2topic/ros2topic/verb/echo.py:112]:
```suggestion
            help='The name of the echoing node; by default, will be a hidden node name')
```

**Review by clalancette — CHANGES_REQUESTED** [review]
I think this is fine.  I've left one wording improvement.

**Comment by fujitatomoya** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/22101/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16356/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1620/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22840/)

**Review by clalancette — APPROVED** [review]

**Comment by fujitatomoya** [comment]
@clalancette thanks for review!

**Pull request merged**